Taskfile.yaml

```diff
@@ -0,0 +1,39 @@
+version: 3
+
+# vars: # cannot be specified here due to https://github.com/go-task/task/issues/2108
+#   NESTED_MODULES: api
+#   API_DIRS: '{{.ROOT_DIR}}/api/core/v1alpha1/...'
+#   MANIFEST_OUT: '{{.ROOT_DIR}}/api/crds/manifests'
+#   CODE_DIRS: '{{.ROOT_DIR}}/cmd/... {{.ROOT_DIR}}/internal/... {{.ROOT_DIR}}/test/... {{.ROOT_DIR}}/api/core/v1alpha1/...'
+#   COMPONENTS: 'project-workspace-operator'
+#   REPO_URL: 'https://github.com/openmcp-project/project-workspace-operator'
+#   GENERATE_DOCS_INDEX: "true"
+
+includes:
+  shared:
+    taskfile: hack/common/Taskfile_controller.yaml
+    flatten: true
+    excludes: [] # put task names in here which are overwritten in this file
+    vars:
+      NESTED_MODULES: api
+      API_DIRS: '{{.ROOT_DIR}}/api/core/v1alpha1/...'
+      MANIFEST_OUT: '{{.ROOT_DIR}}/api/crds/manifests'
+      CODE_DIRS: '{{.ROOT_DIR}}/cmd/... {{.ROOT_DIR}}/internal/... {{.ROOT_DIR}}/test/... {{.ROOT_DIR}}/api/core/v1alpha1/...'
+      COMPONENTS: 'project-workspace-operator'
+      REPO_URL: 'https://github.com/openmcp-project/project-workspace-operator'
+      GENERATE_DOCS_INDEX: "true"
```

Comment:
We should either remove this, or set it up correctly, otherwise it just generates an empty index file.

See https://github.com/openmcp-project/build?tab=readme-ov-file#documentation-index-generation for a short documentation on the feature and https://github.com/openmcp-project/mcp-operator/blob/main/docs/controllers/.docnames for an example of one of the mentioned metadata files.